### ISSUE TITLE
03-Calling-an-API: private api

### ISSUE THREAD

**Issue by Hidatsa** [auth0-samples/auth0-angular-samples#40]
Very nice tutorial, thank you.
 I am getting in above example an error I can't understand. Login with auth0 works fine, I can see profile and public endpint on server works. But for the private endpoint call (Call private button in app) I am getting following error in console: 
```javascript
Error: getaddrinfo ENOTFOUND https https:443
    at errnoException (dns.js:50:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:91:26)
```

and of course the error in app:

```javascript
Response with status: 500 Internal Server Error for URL: http://localhost:3001/api/private
```
I did try the original code from this repo and since it expect some https, I also run the server on port 443 with provided tls (works fine with public). No change for private api call altough.

Any idea where I have to look at?

**Comment by chenkie** [comment]
What happens if you run the API without TLS, any change?

I'm also wondering if it could be something with `AuthHttp` from angular2-jwt, although I don't think it would be since it just wraps Angular's Http. Can you try making a regular Http GET call but add the `access_token` as a header manually?

```ts
import { Http } from '@angular/http';

// ...

public securedPing(): void {
  const headers = new Headers({ 'Authorization': `Bearer ${localStorage.getItem('access_token')}`});
  this.message = '';
  this.authHttp.get(`${this.API_URL}/private`)
    .map(res => res.json())
    .subscribe(
      data => this.message = data.message,
      error => this.message = error
    );
}

```

**Comment by Hidatsa** [comment]
I tried your above suggestion which left me with exactly the same error code. But then I switched to **HS256 algorithm** and I got a more meaningful error message which was pointing to a wrong jwt issuer (problem was that the env file Domain-string contained also the https:// part, this is obviously wrong, it only should contain the identifier string). I switched back to **RS256** again and it works fine.
